src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -203,6 +218,16 @@ private void validateIntervalFilterProperty(String propertyName, String datetime
     }
   }
 
+  private void validateOperationProperty(String propertyName, String operation) {
+    try {
+      OperationEnum.valueOf(operation);
+    } catch (InvalidConfigException e) {
+      throw new InvalidConfigException(
+              String.format("Invalid Query Operation: '%s' value: '%s'. Valid values are query and queryAll.",
```

Comment:
change the string to -
String.format("Invalid Query Operation: '%s'. Valid values are query and queryAll.", operation)